packages/teams-js/src/internal/communication.ts

```diff
@@ -23,6 +22,7 @@ export class Communication {
   public static parentWindow: Window | any;
   public static childWindow: Window;
   public static childOrigin: string;
+  public static validatedDomains: Map<string, boolean> = new Map();
```

Comment:
>validatedDomains [](http://example.com/codeflow?start=16&length=16)

Based on how it looks like this is being used in utils, I _think_ the name `validatedDomains` is confusing, because it implies that the domains contained within "have been validated." In fact, I think it is the case that the domains in here _have been checked for validation_ but have not necessarily actually _been validated_. Is that right? I wouldn't want anyone using this to infer that if the domain was in this object it meant that the domain was okay to use.

Consider a name like `domainsAlreadyCheckedForValidity` or something more catchy.  😜